README.md

```diff
@@ -138,6 +138,24 @@ The context added looks like:
 
 **Note:** `Cloud role name` and `Cloud role instance` shown in the screenshot above, represent service and service instance respectively, and are special fields in Application Insights schema that are used as aggregation units to power certain experiences like Application map.  
 
+### Event Correlation with Service Remoting
+You will have enhanced experience with correlated events across services. If you're using HttpClient to communicate between your services, the correlation is supported implicitly. For Service Remoting, you just need to make some minor changes to your existing code and the correlation will be supported.
+1. For the service invoke the request, change how the proxy is created:

```

Comment:
"For the service invoking/creating/making the request, change how the proxy is created:"